packages/yoga/src/ActionRequirement/web/ActionRequirement.jsx

```diff
@@ -73,15 +73,15 @@ function ActionRequirement(props) {
   defineCompoundComponents();
   return (
     <StyledActionRequirement {...props}>
-      {illustration && <BoxIllustration>{illustration}</BoxIllustration>}
+      {illustration && <BoxIllustration aria-hidden="true">{illustration}</BoxIllustration>}
 
       <Content>
         {titleAsTextDisplay ? (
-          <Text.Display2>{title}</Text.Display2>
+          <Text.Display2 as="h1" aria-label={title} tabIndex={0}>{title}</Text.Display2>
```

Comment:
tabindex="0" is not necessary here, the screen reader will read it correctly

Reply:
Done